tests/themes/run.sh

```diff
@@ -26,6 +27,7 @@ for theme in themes/*/theme.lua; do
   output=$(AWESOME_RC_FILE="$config_file" \
     AWESOME_THEMES_PATH="$PWD/themes" \
     AWESOME_ICON_PATH="$PWD/icons" \
+    VERBOSE=1 \
```

Comment:
Why this commit? Why shouldn't `tests/themes/run.sh` use a `$VERBOSE` env var in the same way that `tests/run.sh` does?